base_local_planner/src/speed_limiters/path_speed_limiter.cpp

```diff
@@ -131,11 +136,10 @@ bool PathSpeedLimiter::calculateLimits(double& max_allowed_linear_vel, double& m
     }
   }
 
-  ROS_DEBUG_THROTTLE(0.2, "Max heading diff: %f", max_heading_diff);
+  ROS_DEBUG("Max heading diff: %f", max_heading_diff);
```

Comment:
why remove the throttle?

Reply:
so throttle uses ros time behind the scenes and for unit testing, I would have to run a roscore and setup ros time etc. Basically, I removed it so that we don't have to make unit test a ros test.

Reply:
makes sense. so is the throttle not necessary at all? i would think it had been added to reduce log spam. if that's not the case seems like it could be removed.

Reply:
it does reduce the log spam but also it is DEBUG already so we don't log it anyways. The only reason throttle was there was to reduce log spam if in case we turn it on.